packages/tools/devtools/devtools-browser-extension/.env

```diff
@@ -0,0 +1 @@
+DEVTOOL_TELEMETRY_TOKEN=abcdefgh-ijkl-mnop-qrst-uvwxyz6ffd9c
```

Comment:
This shouldn't be committed. If you want to provide an example .env file call it something like example.env.